vertx-sql-client/src/main/java/io/vertx/sqlclient/impl/command/StartTxCommand.java

```diff
@@ -0,0 +1,12 @@
+package io.vertx.sqlclient.impl.command;
+
+import io.vertx.sqlclient.TransactionOptions;
+
+public class StartTxCommand<R> extends TxCommand<R> {
+  public TransactionOptions transactionOptions;
+
+  public StartTxCommand(Kind kind, R result, TransactionOptions transactionOptions) {
```

Comment:
rather here directly the access mode and the isolation level as _ TransactionOptions_ is mutable